### PR TITLE
feat: add permission to catalog create and refresh buttion

### PR DESCRIPTION
**Pull request by mario-mui**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/catalog/src/components/AboutCard/AboutCard.test.tsx [hunk 1]
```diff
@@ -276,6 +282,12 @@ describe('<AboutCard />', () => {
       },
     };
 
+    const mockPermissionApi = { authorize: jest.fn() };
+
+    mockPermissionApi.authorize.mockResolvedValueOnce({
+      result: AuthorizeResult.ALLOW,
```
Comment (nuritizra) [inline plugins/catalog/src/components/AboutCard/AboutCard.test.tsx:288]: can you please add a test for DENY

**Review by nuritizra — CHANGES_REQUESTED** [review]

**Review by freben — APPROVED** [review]
Looking good! @nuritizra for final approval and merge?

**Review by vinzscam — APPROVED** [review]
nice!

**Review by nuritizra — APPROVED** [review]

**Comment by mario-mui** [comment]
@nuritizra why this pr not auto merged?

**Pull request merged**